tools/isobuild/test-files.ts

```diff
@@ -20,11 +19,13 @@ export const APP_TEST_FILENAME_REGEXPS = [
 
 // Given a path to a file in an app (relative to the app root
 // directory), is this file a test file?
-export function isTestFilePath(path) {
-  const splitPath = path.split(pathSep);
+export function isTestFilePath(path: string) {
+  // We can remove the || '', but pop function declares a string | undefined
+  // return that is not compatible with regex.test function
+  const fileName = path.split(pathSep).pop() || '';
```

Comment:
Not sure if this is better:
```suggestion
  const splitPath = path.split(pathSep);
  const fileName = splitPath[splitPath.length - 1];
```